tests/integration/test_backup_restore_on_cluster/test_disallow_concurrency.py

```diff
@@ -125,19 +125,29 @@ def test_concurrent_backups_on_same_node():
         .query(f"BACKUP TABLE tbl ON CLUSTER 'cluster' TO {backup_name} ASYNC")
         .split("\t")[0]
     )
-    assert_eq_with_retry(
-        nodes[0],
-        f"SELECT status FROM system.backups WHERE status == 'CREATING_BACKUP' AND id = '{id}'",
-        "CREATING_BACKUP",
+
+    status = (
+        nodes[0]
+        .query(f"SELECT status FROM system.backups WHERE id == '{id}'")
+        .rstrip("\n")
     )
-    assert "Concurrent backups not supported" in nodes[0].query_and_get_error(
+    assert status in ["CREATING_BACKUP", "BACKUP_CREATED"]
+
+    error = nodes[0].query_and_get_error(
         f"BACKUP TABLE tbl ON CLUSTER 'cluster' TO {backup_name}"
     )
+    expected_errors = [
+        "Concurrent backups not supported",
+        f"Backup {backup_name} already exists",
+    ]
+    assert any([expected_error in error for expected_error in expected_errors])
 
     assert_eq_with_retry(
         nodes[0],
-        f"SELECT status FROM system.backups WHERE status == 'BACKUP_CREATED' AND id = '{id}'",
+        f"SELECT status FROM system.backups WHERE id = '{id}'",
         "BACKUP_CREATED",
```

Comment:
Here was the third problem, sometimes a backup's creation takes more than 10 seconds, so I'll increase the timeouts to 10 seconds.